### PR TITLE
fix: avatar size for new design

### PR DESCRIPTION
**Pull request by emmenko**
Size `s` (current)

image

Size `m`

image

### CODE REVIEW

File: packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx [hunk 1]
```diff
@@ -70,17 +69,18 @@ const UserAvatar = (
   const handleMouseOut = useCallback(() => {
     setIsMouseOver(false);
   }, []);
-  const { theme } = useTheme();
+  const { isNewTheme, themedValue } = useTheme();
   return (
     <div onMouseOver={handleMouseOver} onMouseOut={handleMouseOut}>
       <Spacings.Inline alignItems="center">
         <Avatar
+          size={themedValue('s', 'l')}
```
Comment (emmenko) [inline packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx:77]: The correct size

File: packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx [hunk 2]
```diff
@@ -243,20 +243,18 @@ const UserSettingsMenuBody = (props: MenuBodyProps) => {
...
       <div {...props.downshiftProps.getMenuProps()}>
-        <UserSettingsAvatarContainer theme={theme}>
-          <Spacings.Inline
-            scale={theme === 'test' ? 'm' : 'xs'}
-            alignItems="center"
-          >
+        <UserSettingsAvatarContainer isNewTheme={isNewTheme}>
+          <Spacings.Inline scale={themedValue('xs', 'm')} alignItems="center">
             <Avatar
+              size={themedValue('s', 'l')}
```
Comment (emmenko) [inline packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx:252]: ...and here

File: packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx [hunk 3]
```diff
@@ -70,17 +69,18 @@ const UserAvatar = (
   const handleMouseOut = useCallback(() => {
     setIsMouseOver(false);
   }, []);
-  const { theme } = useTheme();
+  const { isNewTheme, themedValue } = useTheme();
```
Comment (emmenko) [inline packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx:72]: Unrelated refactor to use these utilities

File: packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx (hunk 1 above)
Comment (emmenko) [reply packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx:77]: cc @FilPob

**Review by CarlosCortizasCT — APPROVED** [review]
👍

File: packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx (hunk 1 above)
Comment (FilPob) [reply packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx:77]: sorry, I dont know what and where is being changed here

**Review by tdeekens — APPROVED** [review]

File: packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx (hunk 1 above)
Comment (emmenko) [reply packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx:77]: Avatar size

Comment (emmenko) [reply packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx]: Size `l` is too big, I changed it to `m`.

Size `s` (current)

image


Size `m`

image

**Review by kark — APPROVED** [review]
Now looks great imo!

**Review by ddouglasz — APPROVED** [review]
👍🏾

**Review by tdeekens — APPROVED** [review]
